### PR TITLE
005-tilegrid: Add GTP_CHANNEL tilegrid fuzzer

### PR DESCRIPTION
**Pull request by tmichalak**
This PR adds the tilegrid fuzzer for the GTP_CHANNEL tiles.

### CODE REVIEW

File: fuzzers/005-tilegrid/add_tdb.py [hunk 1]
```diff
@@ -111,6 +111,7 @@ def run(fn_in, fn_out, verbose=False):
         ("cfg_int", int_frames, int_words),
         ("monitor_int", int_frames, int_words),
         ("orphan_int_column", int_frames, int_words),
+        ("gtp_channel", 32, 20),
```
Comment (acomodi) [inline fuzzers/005-tilegrid/add_tdb.py:114]: The maximum range of words for each GTP_CHANNEL that I have seen is 22. I should double-check,but the math comes together with 22, as the first word of a GTP_COMMON tile, which is located after two GTP_CHANNELs is 44

File: fuzzers/005-tilegrid/gtp_channel/top.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        params[tile_name] = (site_name, isone)
+
+        attr = "TRUE" if isone else "FALSE"
+        print(
+            '''
+    (* KEEP, DONT_TOUCH, LOC = "{}" *)
+   GTPE2_CHANNEL #(
+        .RXSYNC_OVRD(1'b{})
+    ) gtpe2_channel_{} ();'''.format(
+                site_name, 1 if attr in "TRUE" else 0, site_name))
```
Comment (acomodi) [inline fuzzers/005-tilegrid/gtp_channel/top.py:56]: The `"TRUE"` `"FALSE"` logic can be avoided and directly use `isone`

**Review by acomodi — CHANGES_REQUESTED** [review]
Comments in

File: fuzzers/005-tilegrid/gtp_channel/top.py (hunk 2 above)
Comment (tmichalak) [reply fuzzers/005-tilegrid/gtp_channel/top.py]: Fixed

File: fuzzers/005-tilegrid/add_tdb.py (hunk 1 above)
Comment (tmichalak) [reply fuzzers/005-tilegrid/add_tdb.py]: Changed to 22

**Comment by litghost** [comment]
Please rebase this PR

File: fuzzers/005-tilegrid/Makefile [hunk 3]
```diff
@@ -185,6 +189,7 @@ clean:
 	cd hclk_ioi && $(MAKE) clean
 	cd pcie && $(MAKE) clean
 	cd gtp_common && $(MAKE) clean
+	cd gtp_channel && $(MAKE) clean
```
Comment (litghost) [inline fuzzers/005-tilegrid/Makefile:192]: Need to also add this to `clean_all`.

Comment (tmichalak) [reply fuzzers/005-tilegrid/Makefile:192]: Done

**Pull request merged**